_includes/dc/setup.html

```diff
@@ -43,7 +43,22 @@
 <p>The setup instructions for the Data Carpentry Genomics workshops can be found at <a href="https://datacarpentry.org/genomics-workshop/index.html#setup">the workshop overview site</a>.
 In this workshop, we will run commands and programs on local machines, hence falling within <a href="https://datacarpentry.org/genomics-workshop/index.html#option-b-using-the-lessons-on-your-local-machine">"Option B"</a> of the setup instructions.</p>
 
-<p>We advise participants to wait before installing software and downloading data, as more detailed instructions will be provided shortly.</p>
+<p>All participants are expected to have installed the software described in the setup instructions prior to the workshop.
+If you have trouble installing the software on your machine, please get in touch with the instructors by e-mail or come 30 minutes earlier on the first day of the workshop.
+</p>
+
+<p>Below, we provide additional useful details about the setup.</p>
+
+<p>The programs which will be used during this workshop are easier to install and use on Linux or macOS.
+As a consequence, if using a Mac or Linux machine is not possible, we advise Windows users to install and use the <a href="https://learn.microsoft.com/en-us/windows/wsl/install">Windows Subsystem for Linux (WSL)</a>.
+
+<p>In order to use some programs, it is necessery to have <a href=https://www.java.com/en/download/manual.jsp>Java</a> installed.</p>
+
+<p>Please note that the installation instructions for macOS are meant for Macs with an Intel chip.
```

Comment:
```suggestion
<p>Please note that the setup instructions for macOS are meant for Macs with an Intel chip.
```
So it's clear we're referring to the same instructions as earlier.